src/notation/view/percussionpanel/percussionpanelpadlistmodel.cpp

```diff
@@ -66,7 +65,7 @@ void PercussionPanelPadListModel::init()
 void PercussionPanelPadListModel::addRow()
 {
     for (size_t i = 0; i < NUM_COLUMNS; ++i) {
-        m_padModels.append(new PercussionPanelPadModel(this));
+        m_padModels.append(nullptr);
```

Comment:
why nullptr?

Reply:
This is part of the `isEmptySlot` refactor (fd54f56).

Instead of using `isEmptySlot` we can simply check whether a model is null at a given index in the list. If no model exists, we'll interpret this as an empty slot.

The use of `nullptr` here is to initialise the new row with empty slots.